### DISCUSSION TITLE
Synology-Drive missing from brew.sh but available in brew repository

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#4093] [Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hey there,

i was looking for Synology-Drive on brew.sh, it's not available. I did a 'brew search synology-drive' which is not available aswell. But there is a .rb within the casks on github: https://github.com/Homebrew/homebrew-cask-drivers/blob/master/Casks/synology-drive.rb

I'm just wondering why it's missing on the client and website, but available within github.

Edit: In case it's relevant, i'm using an Apple M1. But i guess this should not effect brew.sh

### Relevant casks

synology-drive

**Comment by SMillerDev** [comment]
Only the default repositories `Homebrew-core` and `Homebrew-cask`show up on the website. In `brew search` it'll show up after you run `brew tap Homebrew/cask-drivers`

**Reply by ghost** [reply]
Oh i missed the drivers part completely. Maybe because Synology Drive is not a driver at all. I can find dropbox, onedrive etc. within the normal casks. So maybe this is just placed wrong, but at least i know where to find it now. If someone else thinks this should not be a driver, feel free to change it, i guess my github skill is not enough to do this :)

**Comment by ghost** [comment]
The more i think about it the more it's located wrong. I don't want to create a bug report for this, but Synology Drive is equivalent to Dropbox/Onedrive/Google Drive, it's a service for synchronizing remote folders no driver that depends on the hardware of the device, so it's clearly located wrong here if i don't miss anything.

The answer was correct though, but i removed the mark so maybe someone reads this.
